src/guide/iroha-2.md

```diff
@@ -3,75 +3,93 @@
 Iroha 2 is a complete re-write of Hyperledger Iroha in Rust. As of writing
 the two projects are developed concurrently.
 
+Here we aspire to outline the differences between two versions of Iroha and
+highlight the new features of Iroha 2. This introduction should be of
+particular interest to those who are already familiar with Iroha but wish
+to upgrade and start using the newer version.
+
+<!-- This list of headlining features goes on. As we are nearing our first
+long-term supported preview release, we'll be updating the guide with
```

Comment:
I've moved this part from the end of the chapter and commented it out because we've already approached the LTS release so I don't think we need to say this.

@appetrosyan is there anything new we want to add to this comparison?

Reply:
Not at this point.